Command/GenerateCommandCommand.php

```diff
@@ -43,7 +43,7 @@ public function configure()
 inside bundles. Provide the bundle name as the first argument and the command
 name as the second argument:
 
-<info>php app/console generate:command AppBundle blog:publish-posts</info>
+<info>php bin/console generate:command AppBundle blog:publish-posts</info>
```

Comment:
All the `app/` to `bin/` changes must be reverted as we support both 2.x and 3.x dir structure.